mindmeld/augmentation.py

```diff
@@ -241,6 +262,167 @@ def __init__(self, batch_size, language, paths, path_suffix, resource_loader):
             "max_length": 60,
         }
 
+    def _download_model(self):
+        logger.info("Downloading paraphrase model from %s", PARAPHRASER_RETAIN_ENTITIES_URL)
+
+        # Make the folder that will contain the model folder
+        if not os.path.exists(EMBEDDINGS_FOLDER_PATH):
+            os.makedirs(EMBEDDINGS_FOLDER_PATH)
+
+        with TqdmUpTo(unit="B", unit_scale=True, miniters=1, desc='') as t:
+            try:
+                urlretrieve(PARAPHRASER_RETAIN_ENTITIES_URL,
+                            PARAPHRASER_FILE_PATH,
+                            reporthook=t.update_to)
+            except ConnectionError as e:
+                logger.error("Model download failed with error: %s", e)
+                return
+        try:
+            with zipfile.ZipFile(PARAPHRASER_FILE_PATH, 'r') as zip_ref:
+                zip_ref.extractall(EMBEDDINGS_FOLDER_PATH)
+        except zipfile.BadZipfile:
+            logger.error("Unable to extract zip file. Try downloading the model again.")
+        return
```

Comment:
Nit: We could remove this last return unless it is intentionally kept to make it clear that None is being returned